www_src/pages/editor/image-editor.jsx

```diff
@@ -0,0 +1,49 @@
+var React = require('react/addons');
+var Binding = require('../../lib/binding.jsx');
+var ColorGroup = require('../../components/color-group/color-group.jsx');
+var Range = require('../../components/range/range.jsx');
+
+var ImageEditor = React.createClass({
+  mixins: [React.addons.LinkedStateMixin, Binding],
+  getInitialState: function () {
+    return {
+      transparency: 70,
+      borderWidth: 0,
+      borderColor: 'transparent'
+    };
+  },
+  render: function () {
+    var style = {
+      opacity: this.state.transparency / 100,
+      borderStyle: 'solid',
+      borderWidth: this.state.borderWidth,
```

Comment:
As for how to do this in a more reliable, consistent way, it requires a decision on the AST stuff we were talking about